packages/react-aria/src/hooks/useARIAButton.ts

```diff
@@ -37,35 +37,37 @@ export function useARIAButton(
       ev.stopPropagation();
       return;
     }
-    if (code === SpacebarKey) {
-      ev.preventDefault();
-      return;
-    }
+    // TODO: Spacebar event should be handled on keyup, but conformance test is breaking
+    // if (code === SpacebarKey) {
+    //   ev.preventDefault();
+    //   return;
+    // }
     // If enter is pressed, activate the button
-    else if (code === EnterKey) {
+    else if (code === EnterKey || code === SpacebarKey) {
       ev.preventDefault();
       ev.currentTarget.click();
     }
   });
 
-  const onKeyupHandler = useEventCallback((ev: React.KeyboardEvent<HTMLElement>) => {
-    const code = getCode(ev);
-    if (typeof onKeyUp === 'function') {
-      onKeyUp(ev);
-    }
-    if (ev.isDefaultPrevented()) {
-      return;
-    }
-    if (disabled && (code === EnterKey || code === SpacebarKey)) {
-      ev.preventDefault();
-      ev.stopPropagation();
-      return;
-    }
-    if (code === SpacebarKey) {
-      ev.preventDefault();
-      ev.currentTarget.click();
-    }
-  });
+  // TODO: this handles Spacebar on keyup, but conformance test is breaking
```

Comment:
Is there a follow-up issue opened to fix conformance tests regarding this?

Reply:
No. there should be a discussion about updating conformance after implementing this https://github.com/microsoft/fluentui/pull/18587

but for the moment, I juist converged `react-aria` to support things as they are at the moment, so nothing breaks.